test/plugins/sensor_demo/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+project(sensor_demo)
```

Comment:
Since this CMakeLists.txt is being included by the top level, , I don't think these lines are necessary.

Reply:
Removed